### PR TITLE
DEVDOCS-4371: Add gift_certificate_id to v2/orders for GET Order Prod…

### PR DESCRIPTION
**Pull request by bc-munjal**
…uct Calls

# [DEVDOCS-4371](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4371)

Reference: 
https://github.com/bigcommerce/bigcommerce/pull/47413
https://github.com/bigcommerce/bigcommerce/pull/47348

## What changed?
Adds a new field "gift_certificate_id" to the response in 

`GET https://api.bigcommerce.com/stores/{store_hash}/v2/orders/{order_id}/products/{product_id} `

@bigcommerce/orders

### CODE REVIEW

File: reference/orders.v2.oas2.yml [hunk 1]
```diff
@@ -3404,6 +3408,10 @@ components:
           type: string
           example: Towel Type 1
           description: The product name that is shown to merchant in Control Panel.
+        gift_certificate_id:
+          type: integer
```
Comment (donald-nguyen-bc) [inline reference/orders.v2.oas2.yml:3412]: 🍹 it should be nullable as well right ?

Comment (slsriehl) [reply reference/orders.v2.oas2.yml:3420]: @bc-munjal ^^ should this be nullable?

cc @bc-tgomez @donald-nguyen-bc

Comment (bc-tgomez) [reply reference/orders.v2.oas2.yml:3420]: About ready to merge these changes but waiting on hearing back on the nullable integer @bc-munjal 👍

File: reference/orders.v2.oas2.yml [hunk 2]
```diff
@@ -3404,6 +3408,10 @@ components:
...
+          example: 52
+          description: Numeric ID of the associated Gift Certificate.
```
Comment (bc-tgomez) [inline reference/orders.v2.oas2.yml:3422]:
```suggestion
          description:  ID of the associated gift certificate.
```

**Review by bc-tgomez — CHANGES_REQUESTED** [review]

File: reference/orders.v2.oas2.yml (hunk 1 above)
Comment (bc-munjal) [reply reference/orders.v2.oas2.yml:3420]: @donald-nguyen-bc : good callout, this needs to be nullable, made that change.

@slsriehl @bc-tgomez thanks for the help, this needs to be nullable, I have made those changes.

File: reference/orders.v2.oas2.yml (hunk 2 above)
Comment (bc-munjal) [reply reference/orders.v2.oas2.yml]: Done this.

**Pull request merged**